### ISSUE TITLE
Swiping feature UI

### ISSUE THREAD

**Issue by joeyda3rd** [Flutter-Buddies/baby-namer#6]
A swiping UI
swipe right or left with notification of accept or reject
undo last swipe

**Comment by lorenzofabro** [comment]
I have a couple of questions: 
- Every option to swipe will contain a name?
- What do yo mean with " **notification** of accept or reject"? Like a little message after you swipe?

**Comment by joeyda3rd** [comment]
Yes. Leave the ability to add more information later, but just a name for now. Or a Gender too, but we can make the swipable cards colors. I picked pink and blue colors that work well together in the wiki, but you can change anything you'd like.
A visualization of accepted or rejected. Can be anything you'd like. Green/red background. Check/X. Just have to let the user know they are saved or rejected.

**Comment by joeyda3rd** [comment]
It may be a good idea to give the user a visualization feedback of the direction they are swiping. For instance, as I'm swiping and before I lift my finger and release I know that when I let go that I will either accept or reject the item because I am getting feedback.   

Pardhu recommends this package https://pub.dev/packages/flutter_tindercard
